cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -62,51 +59,46 @@ type EthereumSub struct {
 }
 
 // NewKeybase create a new keybase instance
-func NewKeybase(validatorMoniker, mnemonic, password string) (keys.Keybase, keys.Info, error) {
-	keybase := keys.NewInMemory()
+func NewKeybase(validatorMoniker, mnemonic, password string) (keyring.Keyring, keyring.Info, error) {
+	kr := keyring.NewInMemory()
 	hdpath := *hd.NewFundraiserParams(0, sdk.CoinType, 0)
-	info, err := keybase.CreateAccount(validatorMoniker, mnemonic, "", password, hdpath.String(), keys.Secp256k1)
+	info, err := kr.NewAccount(validatorMoniker, mnemonic, password, hdpath.String(), hd.Secp256k1)
```

Comment:
// The previous call was:
	// 	.CreateAccount(validatorMoniker, mnemonic, "", password, hdpath.String(), keys.Secp256k1)
	//   name: validatorMoniker
	//   mnemonic: mnemonic
	//   bip39Passwd: ""
	//   encryptPasswd: password
	//   hdPath: hdpath
	//   algo:keys.Secp256k1
	// and CreateAccount is:
	//  CreateAccount(name, mnemonic, bip39Passwd, encryptPasswd, hdPath string, algo SigningAlgo) (Info, error)
	// 
	// The new call is
	//   .NewAccount(validatorMoniker, mnemonic, password, hdpath.String(), hd.Secp256k1)
	//  uid: validatorMoniker
	//  mnemonic: mnemonic
	//  bip39Passwd: password
	//  hdPath: hdpath
	//  algo: hd.Secp256k1
	// for:NewAccount(uid, mnemonic, bip39Passwd, hdPath string, algo SignatureAlgo) (Info, error)
	//  
	// so we used to pass "" to an argument called bip39Passwd, and now we're passing the password
	// to that argument.
	//
	// Was that intentional?

Reply:
It seems like this is a just a call to the cosmos sdk, so I would guess that this change is intentional.